src/components/common/Header.js

```diff
@@ -129,74 +90,128 @@ const useStyles = makeStyles(() => ({
       color: listSelectedTextColor,
     },
   },
-}))
+}));
+
+const breadcrumbs = [
+  { id: 'home', text: 'Home' },
+  { id: 'breadSubItem1', text: 'breadSubItem1' },
+  { id: 'breadSubItem2', text: 'breadSubItem2' },
+  { id: 'breadSubItem3', text: 'breadSubItem3' },
+  { id: 'breadSubItem4', text: 'breadSubItem4' },
+  { id: 'breadSubItem5', text: 'breadSubItem5' },
+  { id: 'breadSubItem6', text: 'breadSubItem6' },
+  { id: 'breadSubItem7', text: 'breadSubItem7' },
+  { id: 'composition2', text: 'Composition 2' },
+];
+
+const listItems = [
+  { label: 'Build', soon: false },
+  { label: 'Test', soon: true },
+  { label: 'Composition', soon: false },
+];
 
 const Header = () => {
   const classes = useStyles();
   const [anchorEl, setAnchorEl] = React.useState(null);
+  const [selected, setSelected] = React.useState('Build');
```

Comment:
@emekauja `setSelected` is not being used or did I miss something?

Reply:
@emekauja I understand but then it should at least be called somewhere so that the next implementation only needs to change the logic and not think about when/where it is used/called. The usage etc is imho part of the UI implementation. The logic behind I agree could be a later implementation